docs/faq.md

```diff
@@ -1,5 +1,47 @@
 # Frequently Asked Questions
 
+## 6-Minute Delays in Attaching Volumes
+
+### What causes 6-minute delays in attaching volumes?
+
+After a node is preempted, pods using persistent volumes may experience a 6+ minute restart delay. This occurs when a volume is not cleanly unmounted, as indicated by `Node.Status.VolumesInUse`. The Attach/Detach controller in `kube-controller-manager` waits for 6 minutes before issuing a force detach. This delay in pod restart can lead to potential data corruption due to unclean mounts.
+
+### What behaviors have been observed that contribute to this issue?
+
+- The CSI driver on the node may get terminated before the workload's volumes are unmounted, leading to unmount failures.
```

Comment:
Suggestion: 

Change: 
> The CSI driver on the node may get terminated before the workload's volumes are unmounted, leading to unmount failures.

To something like:
"The preempted Node's `ebs-csi-node-` pod is terminated before it unmounts all EBS volumes on the node."

In order to stress that the problem lies with the node Daemonset, not the controller deployment, to users who may not be as familiar with the driver.